.github/CODEOWNERS

```
@@ -12,9 +12,12 @@ src/* @mikiquantum @mustermeiszer @branan @NunoAlexandre
 # Changes to chain-specs
 res/* @mikiquantum @mustermeiszer @branan @NunoAlexandre
 
+# Changes to toml files
+*.toml @mikiquantum @mustermeiszer @branan @NunoAlexandre @lemunozm @wischli
+
 # Changes to specific pallets
 ## Connectors related logic
-pallets/connectors/* @NunoAlexandre @offerijns
+pallets/connectors/* @NunoAlexandre @offerijns @wischli
```

Having Jeroen here is not necessirly helpful since he may not have the availaiblity and familiairty with the Rust code at this point to be a blocker for a PR approval. So I would drop Jeroen and add Cosmin and Frederik

```suggestion
pallets/connectors/* @NunoAlexandre @cdamian @wischli @mustermeiszer 
```